pkg/console/metrics/metrics.go

```diff
@@ -0,0 +1,88 @@
+package metrics
```

Comment:
I think I prefer leaving the file as `console_url.go`, as it contains a bunch of helpers only for `console_url`.